notify.plugin.zsh

```diff
@@ -27,15 +29,21 @@ zstyle ':notify:*' error-sound ''
 zstyle ':notify:*' error-icon ''
 zstyle ':notify:*' disable-urgent no
 zstyle ':notify:*' activate-terminal no
+zstyle ':notify:*' always-check-active-window no
 
 unset plugin_dir
 
 # store command line and start time for later
 function zsh-notify-before-command() {
-    local window_id impl
+    local window_id always_check_wid
 
     last_command="$1"
     start_time=$(date "+%s")
+
+    if [[ "$_USING_XDOTOOL" == 1 &&
+        ("$window_id" == "" || always_check_wid == "yes") ]]; then
```

Comment:
shouldn't this whole block fit better in `store-window-id`? there you would not even need to check if using the xdotool version, as that hook is run only in that version.

Reply:
Yup, moved it over there.